scripts/base/protocols/redis/main.zeek

```diff
@@ -0,0 +1,258 @@
+@load base/protocols/conn/removal-hooks
+@load base/frameworks/signatures
+
+@load ./spicy-decls
+
+module Redis;
+
+export {
+	## Log stream identifier.
+	redef enum Log::ID += { LOG };
+
+	## The ports to register Redis for.
+	const ports = {6379/tcp} &redef;
+
+	## Record type containing the column fields of the Redis log.
+	type Info: record {
+		## Timestamp for when the activity happened.
+		ts: time &log;
+		## Unique ID for the connection.
+		uid: string &log;
+		## The connection's 4-tuple of endpoint addresses/ports.
+		id: conn_id &log;
+		## The Redis command.
+		request: Command &log;
+		## The response for the command.
+		response: ServerData &log &optional;
```

Comment:
Given the spec says `command` and `reply`, IMO we should stick with those names rather than introducing `request` and `response`? Maybe shorten `command` to `cmd`.

Could see making `Command.command` field `Command.name`, so it ends up as `cmd.name` in the log.

Reply:
Ah, I see Christian asked the same :-)